frontend/src/components/ExperimentsTable.jsx

```diff
@@ -29,46 +37,111 @@ const ExperimentsTable = (props) => {
+    Cell: (p) => {
+      const lastLogDict = getLastLogDict(p.original);
+      if (logKey === 'elapsed_time') {
+        return <span className="text-right">{lastLogDict.elapsed_time == null ? emptyStr : lastLogDict.elapsed_time.toFixed(2)}</span>;
+      }
+      return <span className="text-right">{lastLogDict[logKey]}</span>;
+    }
+  }));
+
+  const argsList = argKeys.map((argKey) => ({
+    Header: argKey,
+    id: argKey,
+    Cell: (p) => {
+      const { original } = p;
+      const { args } = original;
+      const argDict = {};
+      args.forEach((arg) => {
+        argDict[arg.key] = arg.value;
+      });
+      return <span>{argValue2string(argDict[argKey])}</span>;
+    }
+  }));
+
+  const columns = [
+    {
+      Header: <input
+        type="checkbox"
+        checked={visibleResultCount > 0}
+        style={{ opacity: isPartialSelect ? 0.5 : 1 }}
+        onChange={handleResultsConfigSelectChange}
+      />,
+      Cell: (p) => {
+        const { original } = p;
+        const { id } = original;
+        return (<ToggleResult
+          project={project}
+          result={original}
+          resultConfig={resultsConfig[id]}
+          onResultsConfigSelectUpdate={onResultsConfigSelectUpdate}
+        />);
+      },
+      className: 'text-center',
+      sortable: false,
+      minWidth: 40
+    },
+    {
+      Header: 'id',
+      id: 'result_id',
+      Cell: (p) => {
+        const { original } = p;
+        const { id } = original;
+        return (<Link to={urlForResultDetail(project.id, id)}>{id}</Link>);
+      },
+      className: 'text-center',
+      minWidth: 50
+    },
+    {
+      Header: 'name',
+      id: 'name',
+      Cell: (p) => {
+        const { original } = p;
+        return (<ResultName
+          project={project}
+          result={original}
+          isResultNameAlignRight={globalConfig.isResultNameAlignRight}
+          onResultUpdate={onResultUpdate}
+        />);
+      },
+      minWidth: 250
+    },
+    {
+      Header: 'last logs',
+      columns: logs
+    },
+    {
+      Header: 'args',
+      columns: argsList
+    },
+    {
+      Header: '',
+      Cell: (p) => {
+        const { original } = p;
+        return (<UnregisterButton
+          project={project}
+          result={original}
+          onResultUpdate={onResultUpdate}
+        />);
+      },
+      sortable: false,
+      minWidth: 30
+    }
+  ];
 
   return (
-    <table className="table">
-      <thead>
-        <tr>
-          <th>
-            <input
-              type="checkbox"
-              checked={visibleResultCount > 0}
-              style={{ opacity: isPartialSelect ? 0.5 : 1 }}
-              onChange={handleResultsConfigSelectChange}
-            />
-          </th>
-          <th>id</th>
-          <th>name</th>
-          {logHeaderElems}
-          {argHeaderElems}
-          <th />
-        </tr>
-      </thead>
-      <tbody>
-        {resultRowElems}
-      </tbody>
-    </table>
+    <ReactTable
```

Comment:
I tried to display 100 results, but it only displays 20 results.
Is this because the `defaultPageSize` of `ReactTable` prop is 20?